src/components/sdm/SelectControl.js

```diff
@@ -10,13 +10,15 @@ import {Select} from 'snabbdom-material'
 // import {log} from 'util'
 
 const optionIndex = e => // because children is not a real js array
-  [...e.target.parentNode.children].indexOf(e.target)
+  [...e.target.parentNode.children]
+    .indexOf(e.ownerTarget || e.target)
```

Comment:
Basically we're unable to override `e.currentTarget` in a number of browsers, so we resorted to our own `ownerTarget`. This is because we use event delegation for performance